.gitignore

```diff
@@ -4,3 +4,16 @@ k8s/demo/*test*
 documentation/build/
```

Comment:
It would be great if we could club this under one of the headings (new heading preferred)